website/docs/explorer/operations.mdx

```diff
@@ -219,6 +219,34 @@ datastore_inflight_requests{action="StoreRoleBindings"} 0
 datastore_inflight_requests{action="DeleteAllRoleBindings"} 0
 ```
 
+##### Indexer Writes
+
+**Request Latency:** histogram with the latency of the indexer read requests.
```

Comment:
given it talks about writes, you could replace `read` for `write` in the sentences explain it

Reply:
updated